### ISSUE TITLE
Can not build check client with go modules

### ISSUE THREAD

**Issue by integrii** [Comcast/kuberhealthy#307]
Go modules are throwing a confusing error when you try to build using our checkClient package.  The error is:

```
build github.com/user/cars/cmd/car-checker: cannot load github.com/Comcast/kuberhealthy/pkg/checks/external/checkClient: module github.com/Comcast/kuberhealthy@latest found (v1.0.2), but does not contain package github.com/Comcast/kuberhealthy/pkg/checks/external/checkClient
```

**Comment by integrii** [comment]
It looks like the cause of this is that the package name was `checkclient` and the directory is `checkClient`.

**Comment by integrii** [comment]
Despite a few tries to fix this, all builds are broken as of now.  

```
build github.com/Comcast/kuberhealthy/v2/cmd/kuberhealthy: cannot load github.com/Comcast/kuberhealthy/pkg/checks/external: module github.com/Comcast/kuberhealthy@latest (v1.0.2) found, but does not contain package github.com/Comcast/kuberhealthy/pkg/checks/external
The command '/bin/sh -c go build -v -o kuberhealthy' returned a non-zero code: 1
```

**Comment by integrii** [comment]
The golang site recommends that modules of `v2` and above be placed in a `v2` folder.  This would make our project really messy.  I will continue research on this.

**Comment by integrii** [comment]
Fixed in release v2.1.2.  Please remember to import using the new `v2` paths like this:
`github.com/Comcast/kuberhealthy/v2/pkg/checks/external/checkClient`
